apps/avifdec.c

```diff
@@ -347,6 +347,24 @@ int main(int argc, char * argv[])
     printf("Image details:\n");
     avifImageDump(decoder->image, 0, 0, decoder->progressiveState);
 
+    if (decoder->image->transformFlags & AVIF_TRANSFORM_CLAP) {
+        avifCropRect cropRect;
+        if (avifCropRectFromCleanApertureBox(&cropRect,
+                                             &decoder->image->clap,
+                                             decoder->image->width,
+                                             decoder->image->height,
+                                             &decoder->diag)) {
+            if (cropRect.x != 0 || cropRect.y != 0 || cropRect.width != decoder->image->width ||
+                cropRect.height != decoder->image->height) {
+                // TODO: Implement, see https://github.com/AOMediaCodec/libavif/issues/2427
+                fprintf(stderr, "Warning: Clean Aperture values were ignored, the output image was NOT cropped\n");
```

Comment:
I believe that avifdec.c also ignores the imir and irot properties. Should we also emit warning messages about imir and irot?

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2564.